core/src/main/kotlin/org/evomaster/core/problem/rest/service/fitness/AbstractRestFitness.kt

```diff
@@ -1083,6 +1084,12 @@ abstract class AbstractRestFitness : HttpWsFitness<RestIndividual>() {
 
         analyzeResponseData(fv,individual,actionResults,dto.additionalInfoList)
 
+        // check that excluded fault categories are not present
+        Lazy.assert{
+            DetectedFaultUtils.verifyExcludedCategories(actionResults ,
+                config.getDisabledSecurityOracleCodesList() as List<FaultCategory>
+            )
+        }
```

Comment:
might not be best place for this (unless we duplicate it), as it would not apply to BlackBox isn't it? also, we should have it generic in a way that we can check also for other types, such as GraphQL and RPC (so maybe somewhere in Enterprise* abstract classes). 
Also, checks for skipping should apply for all codes, not just security, ie, 500 http and schema validation